src/StreamJsonRpc.Tests/FullDuplexStream.cs

```diff
@@ -32,13 +30,15 @@ public class FullDuplexStream : Stream
         /// <summary>
         /// The stream to write to.
         /// </summary>
+#pragma warning disable CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable.
         private FullDuplexStream other;
+#pragma warning restore CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable.
```

Comment:
❔ Why is this one excluded?

Reply:
This field creates a circular reference loop with another instance of this type. So it's impossible for this field to be set in the constructor.